Stripe/NSError+STPCustomerContext.m

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
I'd prefer to move this (and all other stripe-specific errors) into StripeError so that we can see them all in one place.